Applications/FileIO/GocadIO/Property.h

```diff
@@ -0,0 +1,51 @@
+/**
+ *
+ * @copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/LICENSE.txt
+ */
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+
+#include <logog/include/logog.hpp>
+
+namespace FileIO
+{
+namespace Gocad
+{
+struct Property final
+{
+    std::size_t _property_id{};
```

Comment:
The default initialization is not needed, it's done anyway.
```suggestion
    std::size_t _property_id;
```

Reply:
Suggested by clang-tidy.